chart/values.yaml

```diff
@@ -343,6 +343,7 @@ deleteIndexes:
 
 
 metricsCollector:
```

Comment:
rename to queueMetricsCollector?

Reply:
Done in https://github.com/huggingface/datasets-server/pull/1658/commits/d75b8cde903ca9b317e9f7c4bcb50518e0a130c8